### ISSUE TITLE
Version `0.2.2` of `@rspc/client` introduces failing to resolve package

### ISSUE THREAD

**Issue by johann-crabnebula** [oscartbeaumont/rspc#295]
# Version `0.2.2` of `@rspc/client` introduces failing to resolve package

When trying to import from `@rspc/client` like the following:

```typescript
import { RSPCError } from "@rspc/client";
```

In dev:
```bash
✘ [ERROR] Failed to resolve entry for package "@rspc/client". The package may have incorrect main/module/exports specified in its package.json. [plugin vite:dep-scan]
```

On build:
```bash
[commonjs--resolver] Failed to resolve entry for package "@rspc/client". The package may have incorrect main/module/exports specified in its package.json.
```

Could confirm the problem for `0.2.2` and `0.2.3`. `0.2.1` is last version that works for me.

**Comment by johann-crabnebula** [comment]
Update:

I seem to have the same problem with  `0.2.3` of `@rspc/solid-query`, throws the following error:

```bash
[commonjs--resolver] Failed to resolve entry for package "@rspc/query-core". The package may have incorrect main/module/exports specified in its package.json.
```

**Comment by oscartbeaumont** [comment]
Just published `0.2.4` which should hopefully fix it. Thanks @Brendonovich  #296 .

**Comment by johann-crabnebula** [comment]
Looks fixed on my end now! Thanks guys!
